lib/fprotect/Kconfig

```diff
@@ -41,6 +41,7 @@ config FPROTECT_BLOCK_SIZE
 	default NRF_MPU_FLASH_REGION_SIZE if HAS_HW_NRF_MPU
 	default NRF_BPROT_FLASH_REGION_SIZE if HAS_HW_NRF_BPROT
 	default NRF_ACL_FLASH_REGION_SIZE if HAS_HW_NRF_ACL
+	default 0x400 if SOC_SERIES_NRF54LX
```

Comment:
How about something like `NRF_RRAM_REGION_SIZE_GRANULARITY` instead of a magic number?